static/app-strings.json

```diff
@@ -2040,5 +2040,11 @@
   "Tip Creators": "Tip Creators",
   "Only select creators can receive tips at this time": "Only select creators can receive tips at this time",
   "The payment will be made from your saved card": "The payment will be made from your saved card",
+  "Trending for #Art": "Trending for #Art",
+  "Trending for #Education": "Trending for #Education",
+  "Trending for #Technology": "Trending for #Technology",
+  "Watch content and earn more Credits for each level unlocked! 10 views required for level 1 (Current Score: 0). Only up to 10 views per day count.": "Watch content and earn more Credits for each level unlocked! 10 views required for level 1 (Current Score: 0). Only up to 10 views per day count.",
```

Comment:
These `Trending for xxx` entries is a double-entry bug in the localization code.  It can be discarded for now.